okhttp/src/main/java/io/grpc/transport/okhttp/OkHttpClientTransport.java

```diff
@@ -48,7 +50,6 @@
 import io.grpc.Status.Code;
 import io.grpc.transport.ClientStreamListener;
 import io.grpc.transport.ClientTransport;
-
```

Comment:
I was surprised why the checkStyle didn't catch it and found we didn't add "<property name="separateLineBetweenGroups" value="true"/>", but even after I added such property, it still couldn't be caught, then I figured out our "io.grpc.*" are considered as third party imports.

Seems we have to manually maintain this style. 

Done.